### ISSUE TITLE
Layout/MultilineBlockLayout triggering if single line would result in exactly 121 characters

### ISSUE THREAD

**Issue by KNejad** [rubocop-hq/rubocop#8189]
This scope is giving a Layout/MultilineBlockLayout cop warning:

```
scope :xxxxxxxxxxxxxxx, lambda {
  |yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy, zzzzzzzzzzzz, wwwwwwwwwwwwwwwwwwwwwwwwww|
}
```

If I add a single character to any of the variables then the warning goes away, this is likely due to (I'm guessing) the cop not triggering if the single line version would be too long (over 120 characters).

The problem is, that the single line version is already 121 characters.

It seems when doing the calculations for how long a single line would be, it's forgetting that I would need to add a space after the `{`. This is confirmed when I add a trailing space, then the warning goes away (however another warning appears)

--------

## Expected behavior

This should not give any warnings:
```
scope :xxxxxxxxxxxxxxx, lambda {
  |yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy, zzzzzzzzzzzz, wwwwwwwwwwwwwwwwwwwwwwwwww|
}
```

This should give a warning:
```
scope :xxxxxxxxxxxxxxx, lambda {
  |yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy, zzzzzzzzzzzz, wwwwwwwwwwwwwwwwwwwwwwwwwww|
}
```
(difference being one extra `w`)

## Actual behavior

Both examples gave a warning.

## Steps to reproduce the problem

Simply paste the example scope into a script and run rubocop.

Try this as well to confirm that it gives a line length warning which is why I had to break it up into 2 lines:

```
scope :xxxxxxxxxxxxxxx, lambda { |yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy, zzzzzzzzzzzz, wwwwwwwwwwwwwwwwwwwwwwwwww|
}
```

## RuboCop version

```
$ rubocop -V
0.86.0 (using Parser 2.7.1.0, rubocop-ast 0.0.3, running on ruby 2.7.0 x86_64-linux)
```

**Comment by KNejad** [comment]
Just a heads up I have created a fix for this: #8203
